src/mixins/filing-template-mixin.ts

```diff
@@ -270,6 +270,90 @@ export default class FilingTemplateMixin extends DateMixin {
     return filing
   }
 
+  /**
```

Comment:
As above, I copied this from another filing. It needs cleanup.

Ditto for "parse" below.